src/rpdk/core/contract/suite/handler_update.py

```diff
@@ -46,5 +51,46 @@ def contract_update_list_success(updated_resource, resource_client):
     # should be able to use the created model
     # to read since physical resource id is immutable
     _create_request, _created_model, _update_request, updated_model = updated_resource
+    test_primary_identifier_not_updated(
+        _created_model, updated_model, resource_client.get_primary_identifier()
+    )
     models = test_list_success(resource_client, updated_model)
     assert updated_model in models
+
+
+@pytest.mark.update
+def contract_update_invalid_property(resource_client):
+    if resource_client.has_read_create_property:
```

Comment:
not sure what this means

Reply:
This test updates the resource with createOnly/readOnly property and expects it to fail.